design/docs/stale-txid-promotion-candidate-pruning.md

```
@@ -0,0 +1,41 @@
+# Stale TxID Promotion Candidate Pruning
+
+## Status
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)